shell.c

```diff
@@ -75,6 +77,35 @@ main(int argc, char *argv[])
   int cycle_offset = 0;
   int num_cycles = CYCLES_PER_TICK / 2;
 
+  // The surface contained by the window
+
+  // Initialize SDL
+  if (SDL_Init(SDL_INIT_VIDEO | SDL_INIT_TIMER | SDL_INIT_JOYSTICK) < 0)
+    {
+      printf("SDL could not initialize! SDL_Error: %s\n", SDL_GetError());
+    }
+  else
+    {
+      // Create window
+      window = SDL_CreateWindow("Space Invaders Emulator",
+                                SDL_WINDOWPOS_UNDEFINED,
+                                SDL_WINDOWPOS_UNDEFINED, SCREEN_WIDTH * 2,
+                                SCREEN_HEIGHT * 2, SDL_WINDOW_RESIZABLE);
```

Comment:
What is the *2 for? Is that just to make a larger screen?

Reply:
yea, just to make it bigger.  it might actually be scuffed though because I didn't add the resizing to it.  Might have to change that.

Reply:
Resizing seems like a nice to have at this point. I'm concerned trying to handle resizing too early could have downstream consequences if not done really carefully (e.g. index out of bounds issues etc).